node-js-web-server/server070.js

```diff
@@ -0,0 +1,30 @@
+const http = require('http');
+
+http.createServer(function (request, response) {
+   console.log('request url: ', request.url);
+}).listen(8080);
+console.log('Server is running at http://localhost:8080');
+
+// so we have this files structure:
```

Comment:
Our project has this file structure:
```
 ++public
    --index.html
    --products.html
    --about.html
    --404.html
    --forrest1.png
    --forrest2.png
    --forrest3.png
    --style.css
 --server.js
 --package.json
 --README.md
```
When users visit a certain URL, our server should respond with the correct page -- `http://localhost:8080/products` should get the response `./public/products.html`.
Create a variable called `filePath` and set it equal to `'./public' + request.url`.

Reply:
Just noticed that the code is using `var` rather than `let` or `const`. We could change this in the future.

Reply:
> Just noticed that the code is using `var` rather than `let` or `const`. We could change this in the future.

I didn't notice it. I only change the module with `const`. I think we can change it right away.

Reply:
Okay! Changing that along with the edited test descriptions I suggested would be really great. Please just submit another PR whenever you can :+1: :+1: